github-orgs/kubeflow/org.yaml

```diff
@@ -1217,3 +1217,12 @@ orgs:
             privacy: closed
             repos:
               xgboost-operator: write
+          kubeflow-outreach-committee:
```

Comment:
Can you keep this list in the alphabetical order and add this team before `kubeflow-steering-committee` ?